### ISSUE TITLE
Laravel 5.4 Support

### ISSUE THREAD

**Issue by strebl** [Codeception/Codeception#3950]
#### What are you trying to achieve?
I'm trying to use Codeception (v2.2.8) with Laravel 5.4.

#### What do you get instead?

I can't run my test suite because `Illuminate\Support\ClassLoader`, with gets referenced in the [Laravel5 module](https://github.com/Codeception/Codeception/blob/2.2/src/Codeception/Module/Laravel5.php#L224), isn't available in Laravel 5.4

#### Exception Trace 
```bash

  [Symfony\Component\Debug\Exception\FatalThrowableError]
  Class 'Illuminate\Support\ClassLoader' not found


Exception trace:
 () at /Users/strebel/Dropbox/bluesheep/Code/pi.strebl.ch/vendor/codeception/codeception/src/Codeception/Module/Laravel5.php:224
 Codeception\Module\Laravel5->registerAutoloaders() at /Users/strebel/Dropbox/bluesheep/Code/pi.strebl.ch/vendor/codeception/codeception/src/Codeception/Module/Laravel5.php:155
 Codeception\Module\Laravel5->_initialize() at /Users/strebel/Dropbox/bluesheep/Code/pi.strebl.ch/vendor/codeception/codeception/src/Codeception/SuiteManager.php:81
 Codeception\SuiteManager->initialize() at /Users/strebel/Dropbox/bluesheep/Code/pi.strebl.ch/vendor/codeception/codeception/src/Codeception/Codecept.php:181
 Codeception\Codecept->runSuite() at /Users/strebel/Dropbox/bluesheep/Code/pi.strebl.ch/vendor/codeception/codeception/src/Codeception/Codecept.php:152
 Codeception\Codecept->run() at /Users/strebel/Dropbox/bluesheep/Code/pi.strebl.ch/vendor/codeception/codeception/src/Codeception/Command/Run.php:360
 Codeception\Command\Run->runSuites() at /Users/strebel/Dropbox/bluesheep/Code/pi.strebl.ch/vendor/codeception/codeception/src/Codeception/Command/Run.php:287
 Codeception\Command\Run->execute() at /Users/strebel/Dropbox/bluesheep/Code/pi.strebl.ch/vendor/symfony/console/Command/Command.php:262
 Symfony\Component\Console\Command\Command->run() at /Users/strebel/Dropbox/bluesheep/Code/pi.strebl.ch/vendor/symfony/console/Application.php:826
 Symfony\Component\Console\Application->doRunCommand() at /Users/strebel/Dropbox/bluesheep/Code/pi.strebl.ch/vendor/symfony/console/Application.php:189
 Symfony\Component\Console\Application->doRun() at /Users/strebel/Dropbox/bluesheep/Code/pi.strebl.ch/vendor/symfony/console/Application.php:120
 Symfony\Component\Console\Application->run() at /Users/strebel/Dropbox/bluesheep/Code/pi.strebl.ch/vendor/codeception/codeception/src/Codeception/Application.php:103
 Codeception\Application->run() at /Users/strebel/Dropbox/bluesheep/Code/pi.strebl.ch/vendor/codeception/codeception/codecept:34
```

### Details

* Codeception version: **2.2.8**
* PHP Version: **7.1.0**
* Operating System: **MacOS 10.12.3**
* Installation type: **Composer**
* Suite configuration:

```yml
# Codeception Test Suite Configuration

# suite for functional (integration) tests.
# emulate web requests and make application process them.
# Include one of framework modules (Symfony2, Yii2, Laravel4) to use it.

class_name: FunctionalTester
modules:
    enabled: [Filesystem, FunctionalHelper, Laravel5, Asserts]
    config:
      Laravel5:
        environment_file: .env.testing
        cleanup: false
```

**Comment by janhenkgerritsen** [comment]
I just merged #3952, that should fix all issues for the Laravel 5.4 release. 

We'll try to release a new version of Codeception ASAP, but until then you can test out this update by specifying `2.2.x-dev` as the version for Codeception in your composer.json file.

**Comment by strebl** [comment]
It works, thanks!
